test/seeds/orders.js

```diff
@@ -46,6 +46,46 @@ const seed = async () => {
       owner: '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY',
     },
   ])
+
+  await client('recipes').insert([
+    {
+      id: '36345f4f-0000-42e2-83f9-79e2e195e000',
+      external_id: '045240',
+      name: 'Magical Part 1',
+      image_attachment_id: '00000000-0000-1000-8000-000000000000',
+      material: 'material',
+      alloy: 'alloy',
+      latest_token_id: 1,
```

Comment:
is having a seeded `token_id` a good idea? Can't guarantee it's correct on the chain so any test trying to burn this recipe will likely fail

Reply:
true! and it's important which type but for anything to work we need a recipe so based on that we can assume that token_id will recipe as without recipe token we can not create an order or to be fair anything else.

Later will be updated to call few times runProcess so it can be read from db, executed against real node in github action and then we can even more expand on testing. When we come to the point where we want to test the whole scenariio e.g. from recipe to build or even part

Reply:
So this mostly depends on what these tests are. By our usual definition of integration tests these would test functional flows all the way down to the chain. I think that is useful, and at that point isolated database seeds won't be that helpful. Instead we'll need some pre-condition apparatus that sets up the db and chain into a known state prior to an actual test running. What I don't understand is why we're expanding the seed data in the integration tests. I'm happy to have a dev seed but that shouldn't be here. Are any of our tests using these changes? I'm assuming not as no tests have changed